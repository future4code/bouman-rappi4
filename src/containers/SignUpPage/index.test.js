import React from 'react'
import renderer from "react-test-renderer";
import {SignUpPage} from './index';

describe ('teste componente Loader', () => {
    it('renders correctly', () => {
        const tree = renderer
            .create(<SignUpPage></SignUpPage>)
            .toJSON();
        expect(tree).toMatchSnapshot();
    })
})