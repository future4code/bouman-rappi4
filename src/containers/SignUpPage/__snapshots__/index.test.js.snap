// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`teste componente Loader renders correctly 1`] = `
<div>
  <header
    className="sc-jKJlTe jlVaSr"
  >
    <img
      className="sc-ckVGcZ hArevR"
      src="back.png"
    />
    <p />
  </header>
  <header
    className="sc-EHOje eiriIZ"
  >
    <img
      className="sc-dnqmqq knXpJk"
      src="logo-future-eats-invert.png"
    />
  </header>
  <form
    className="sc-bZQynM bmtTsa"
    onSubmit={[Function]}
  >
    <h3>
      Cadastre-se
    </h3>
    <div
      className="MuiFormControl-root-1 sc-jzJRlG mhIOd"
      pattern="[A-Za-z]"
    >
      <label
        className="MuiFormLabel-root-16 MuiFormLabel-required-21 MuiInputLabel-required-9 MuiInputLabel-root-5 MuiInputLabel-formControl-10 MuiInputLabel-animated-13 MuiInputLabel-outlined-15"
        data-shrink={false}
      >
        Nome 
        <span
          className="MuiFormLabel-asterisk-22"
        >
           *
        </span>
      </label>
      <div
        className="MuiInputBase-root-36 MuiOutlinedInput-root-23 MuiInputBase-formControl-37"
        onClick={[Function]}
      >
        <fieldset
          aria-hidden={true}
          className="MuiPrivateNotchedOutline-root-53 MuiOutlinedInput-notchedOutline-30"
          style={
            Object {
              "paddingLeft": 8,
            }
          }
        >
          <legend
            className="MuiPrivateNotchedOutline-legend-54"
            style={
              Object {
                "width": 0.01,
              }
            }
          >
            <span
              dangerouslySetInnerHTML={
                Object {
                  "__html": "&#8203;",
                }
              }
            />
          </legend>
        </fieldset>
        <input
          aria-invalid={false}
          className="MuiInputBase-input-46 MuiOutlinedInput-input-31"
          disabled={false}
          name="name"
          onBlur={[Function]}
          onChange={[Function]}
          onFocus={[Function]}
          placeholder="Nome e Sobrenome"
          required={true}
          type="text"
        />
      </div>
    </div>
    <div
      className="MuiFormControl-root-1 sc-jzJRlG mhIOd"
      pattern="[A-Za-^([a-zA-Z0-9_-.]+)@([a-zA-Z0-9_-.]+).([a-zA-Z]{2,5})$]{3,}"
    >
      <label
        className="MuiFormLabel-root-16 MuiFormLabel-required-21 MuiInputLabel-required-9 MuiInputLabel-root-5 MuiInputLabel-formControl-10 MuiInputLabel-animated-13 MuiInputLabel-outlined-15"
        data-shrink={false}
      >
        E-mail: 
        <span
          className="MuiFormLabel-asterisk-22"
        >
           *
        </span>
      </label>
      <div
        className="MuiInputBase-root-36 MuiOutlinedInput-root-23 MuiInputBase-formControl-37"
        onClick={[Function]}
      >
        <fieldset
          aria-hidden={true}
          className="MuiPrivateNotchedOutline-root-53 MuiOutlinedInput-notchedOutline-30"
          style={
            Object {
              "paddingLeft": 8,
            }
          }
        >
          <legend
            className="MuiPrivateNotchedOutline-legend-54"
            style={
              Object {
                "width": 0.01,
              }
            }
          >
            <span
              dangerouslySetInnerHTML={
                Object {
                  "__html": "&#8203;",
                }
              }
            />
          </legend>
        </fieldset>
        <input
          aria-invalid={false}
          className="MuiInputBase-input-46 MuiOutlinedInput-input-31 MuiInputBase-inputType-49"
          disabled={false}
          name="email"
          onBlur={[Function]}
          onChange={[Function]}
          onFocus={[Function]}
          placeholder="email@email.com"
          required={true}
          type="email"
        />
      </div>
    </div>
    <div
      className="MuiFormControl-root-1 sc-jzJRlG mhIOd"
    >
      <label
        className="MuiFormLabel-root-16 MuiFormLabel-required-21 MuiInputLabel-required-9 MuiInputLabel-root-5 MuiInputLabel-formControl-10 MuiInputLabel-animated-13 MuiInputLabel-outlined-15"
        data-shrink={false}
      >
        CPF
        <span
          className="MuiFormLabel-asterisk-22"
        >
           *
        </span>
      </label>
      <div
        className="MuiInputBase-root-36 MuiOutlinedInput-root-23 MuiInputBase-formControl-37"
        onClick={[Function]}
      >
        <fieldset
          aria-hidden={true}
          className="MuiPrivateNotchedOutline-root-53 MuiOutlinedInput-notchedOutline-30"
          style={
            Object {
              "paddingLeft": 8,
            }
          }
        >
          <legend
            className="MuiPrivateNotchedOutline-legend-54"
            style={
              Object {
                "width": 0.01,
              }
            }
          >
            <span
              dangerouslySetInnerHTML={
                Object {
                  "__html": "&#8203;",
                }
              }
            />
          </legend>
        </fieldset>
        <input
          aria-invalid={false}
          className="MuiInputBase-input-46 MuiOutlinedInput-input-31 MuiInputBase-inputType-49"
          disabled={false}
          name="cpf"
          onBlur={[Function]}
          onChange={[Function]}
          onFocus={[Function]}
          placeholder="000.000.000-00"
          required={true}
          type="number"
        />
      </div>
    </div>
    <div
      className="MuiFormControl-root-1 sc-jzJRlG mhIOd"
    >
      <label
        className="MuiFormLabel-root-16 MuiFormLabel-required-21 MuiInputLabel-required-9 MuiInputLabel-root-5 MuiInputLabel-formControl-10 MuiInputLabel-animated-13 MuiInputLabel-outlined-15"
        data-shrink={false}
      >
        Senha
        <span
          className="MuiFormLabel-asterisk-22"
        >
           *
        </span>
      </label>
      <div
        className="MuiInputBase-root-36 MuiOutlinedInput-root-23 MuiInputBase-formControl-37"
        onClick={[Function]}
      >
        <fieldset
          aria-hidden={true}
          className="MuiPrivateNotchedOutline-root-53 MuiOutlinedInput-notchedOutline-30"
          style={
            Object {
              "paddingLeft": 8,
            }
          }
        >
          <legend
            className="MuiPrivateNotchedOutline-legend-54"
            style={
              Object {
                "width": 0.01,
              }
            }
          >
            <span
              dangerouslySetInnerHTML={
                Object {
                  "__html": "&#8203;",
                }
              }
            />
          </legend>
        </fieldset>
        <input
          aria-invalid={false}
          className="MuiInputBase-input-46 MuiOutlinedInput-input-31 MuiInputBase-inputType-49"
          disabled={false}
          name="password"
          onBlur={[Function]}
          onChange={[Function]}
          onFocus={[Function]}
          placeholder="Mínimo 6 caracteres"
          required={true}
          type="password"
        />
      </div>
    </div>
    <div
      className="MuiFormControl-root-1 sc-jzJRlG mhIOd"
    >
      <label
        className="MuiFormLabel-root-16 MuiFormLabel-required-21 MuiInputLabel-required-9 MuiInputLabel-root-5 MuiInputLabel-formControl-10 MuiInputLabel-animated-13 MuiInputLabel-outlined-15"
        data-shrink={false}
      >
        Confirmar
        <span
          className="MuiFormLabel-asterisk-22"
        >
           *
        </span>
      </label>
      <div
        className="MuiInputBase-root-36 MuiOutlinedInput-root-23 MuiInputBase-formControl-37"
        onClick={[Function]}
      >
        <fieldset
          aria-hidden={true}
          className="MuiPrivateNotchedOutline-root-53 MuiOutlinedInput-notchedOutline-30"
          style={
            Object {
              "paddingLeft": 8,
            }
          }
        >
          <legend
            className="MuiPrivateNotchedOutline-legend-54"
            style={
              Object {
                "width": 0.01,
              }
            }
          >
            <span
              dangerouslySetInnerHTML={
                Object {
                  "__html": "&#8203;",
                }
              }
            />
          </legend>
        </fieldset>
        <input
          aria-invalid={false}
          className="MuiInputBase-input-46 MuiOutlinedInput-input-31 MuiInputBase-inputType-49"
          disabled={false}
          name="confirmPassword"
          onBlur={[Function]}
          onChange={[Function]}
          onFocus={[Function]}
          placeholder="Confirme a senha anterior"
          required={true}
          type="password"
        />
      </div>
    </div>
    <button
      className="MuiButtonBase-root-81 MuiButton-root-55 MuiButton-text-57 MuiButton-flat-60 sc-htoDjs iNysJe"
      disabled={false}
      onBlur={[Function]}
      onContextMenu={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      onMouseDown={[Function]}
      onMouseLeave={[Function]}
      onMouseUp={[Function]}
      onTouchEnd={[Function]}
      onTouchMove={[Function]}
      onTouchStart={[Function]}
      tabIndex="0"
      type="submit"
    >
      <span
        className="MuiButton-label-56"
      >
        Criar
      </span>
      <span
        className="MuiTouchRipple-root-84"
      />
    </button>
  </form>
</div>
`;
